test/integration/driver-virtualbox.bats

```diff
@@ -316,6 +322,15 @@ findCPUCount() {
   [ "$status" -eq 0  ]
 }
 
+@test "$DRIVER: upgrade should work" {
+  buildMachineWithOldIsoCheckUpgrade
+  [ "$status" -eq 0 ]
```

Comment:
Perhaps using `docker version` with some grep magic to make sure it's the version?